### ISSUE TITLE
WER stuck at 0.3 when fine tuned with vaccum noise 

### ISSUE THREAD

**Issue by bagavi** [NVIDIA/NeMo#858]
_Goal_: Fine tuning Quartznet 15x5 with test_clean_100 plus vacuum noise (10db snr).

_Setting_: I am using speech2text.py script to train with a single small GPU with `batch size=8, lr = 1e-4, warm_ratio=0.02, num_epochs = 200` and rest of the parameters are default. I have attached the screen shot of WandB board. ![weights and biases board](https://user-images.githubusercontent.com/854366/87704220-ec0ff000-c750-11ea-88b3-06d0d6556a4b.png)

_Problem:_ The WER is stuck to 0.3 (with noise) and 0.2 (without noise) from epoch 4 to 14.

_Questions_:
1. Is this expected since I am using only batch size of 8 and its only run for 14 epochs. 
2. Should I reduce my number of epochs from 200 to 10 or 20?
3. Should I reduce my learning rate to account for batch size of 8?

Is it possible to share the WandB boards for the fine-tuning from your latest papers?

Thanks in advance and congratulations for this awesome work :)

**Comment by okuchaiev** [comment]
I'd recommend you increase warmup_ratio to 0.12.
Also, your batch size is small - we never tried such small bs, but this means that you have to use smaller bs than in our experiments.

**Comment by bagavi** [comment]
Could you clarify “you have to use smaller *bs* than in our experiment”. Do you mean smaller lr(learning rate)?

Also, what is the reason behind .12 warmup ratio for fine tuning vs .02 warmup ratio for training from scratch?

Thanks!
